website/docs/troubleshooting.md

```diff
@@ -48,6 +48,16 @@ You need to Provide the correct socket path. By default in Portal `CONTAINER_RUN
 If Your container runtime is `containerd` then you have to change the `CONTAINER_RUNTIME` to `containerd` and `SOCKET_PATH` to `/var/run/containerd/containerd.sock`.
 You can find these in tune faults part of the tune chaos experiment page.
 
+### The probe only accepts values in `ns, us, ms, m, s, or h`. But Why do experiments fail with `must be of type integer`?
+
+In the logs of the Helper pod, you may face these error logs below.
+
+```shell
+{"mainLogs":"W1003 08:59:55.273647       1 client_config.go:552] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.\n2023/10/03 08:59:55 Error Creating Resource : ChaosEngine.litmuschaos.io 'pod-network-loss-h6srhrls' is invalid: [spec.experiments[0].spec.probe[0].runProperties.interval: Invalid value: 'string': spec.experiments[0].spec.probe[0].runProperties.interval in body must be of type integer: 'string', spec.experiments[0].spec.probe[0].runProperties.probeTimeout: Invalid value: 'string': spec.experiments[0].spec.probe[0].runProperties.probeTimeout in body must be of type integer: 'string']\n"}
+```
+
+It's because your cluster using old CRD manifest files. So you need to delete all Litmus' CRDs and reinstall them. CRDs definition is in the Chaos Infrastructure yaml file.
```

Comment:
Suggestion: You can also add a link to the latest crd manifest files.

Reply:
Updated it :) @Saranya-jena

Reply:
And one question.
I think we don't have to manage `chaoscenter/manifests` [folder](https://github.com/litmuschaos/litmus/tree/master/chaoscenter/manifests).

Because It is duplicated with mkdocs folder. Also, The crds in the chaoscenter/manifests are outdated.
How do you think?

Reply:
In mkdocs we are storing the release manifests, but in chaoscenter/manifests we are storing the ci manifest. Do you have any suggestions regarding this? 🤔

Reply:
I will then raise a PR to update the outdated ChaosEngine CRD. Is it okay @Saranya-jena ?

Reply:
Yes that would be great @namkyu1999 !